content/zh/docs/reference/config-api/kube-scheduler-config.v1beta2.md

```diff
@@ -0,0 +1,1734 @@
+weight 值必须是正整数。
+</td>
+</tr>
+<tr><td><code>bindVerb</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--
+   Verb for the bind call, empty if not supported. This verb is appended to the URLPrefix when issuing the bind call to extender.
+If this method is implemented by the extender, it is the extender's responsibility to bind the pod to apiserver. Only one extender
+can implement this function.
+   -->
+   bind 调用所使用的动词，如果不支持过滤操作则为空。
+此动词会在向扩展模块发送 bind 调用时追加到 urlPrefix 后面。
+如果扩展模块实现了此方法，扩展模块要负责将 Pod 绑定到 API 服务器。
+只有一个扩展模块可以实现此函数。
+</td>
+</tr>
+<tr><td><code>enableHTTPS</code> <B><!--[Required]-->[必需]</B><br/>
+<code>bool</code>
+</td>
+<td>
+   <!--
+   EnableHTTPS specifies whether https should be used to communicate with the extender
+   -->
+   此字段设置是否需要使用 HTTPS 来与扩展模块通信。
+</td>
+</tr>
+<tr><td><code>tlsConfig</code> <B><!--[Required]-->[必需]</B><br/>
+<a href="#kubescheduler-config-k8s-io-v1beta2-ExtenderTLSConfig"><code>ExtenderTLSConfig</code></a>
+</td>
+<td>
+   <!--
+   TLSConfig specifies the transport layer security config
+   -->
+   此字段设置传输层安全性（TLS）配置。
+</td>
+</tr>
+<tr><td><code>httpTimeout</code> <B><!--[Required]-->[必需]</B><br/>
+<a href="https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"><code>meta/v1.Duration</code></a>
+</td>
+<td>
+   <!--
+   HTTPTimeout specifies the timeout duration for a call to the extender. Filter timeout fails the scheduling of the pod. Prioritize
+timeout is ignored, k8s/other extenders priorities are used to select the node.
+   -->
+   此字段给出扩展模块功能调用的超时值。filter 操作超时会导致 Pod 无法被调度。
+prioritize 操作超时会被忽略，Kubernetes 或者其他扩展模块所给出的优先级值
+会被用来选择节点。
+</td>
+</tr>
+<tr><td><code>nodeCacheCapable</code> <B><!--[Required]-->[必需]</B><br/>
+<code>bool</code>
+</td>
+<td>
+   <!--
+   NodeCacheCapable specifies that the extender is capable of caching node information,
+so the scheduler should only send minimal information about the eligible nodes
+assuming that the extender already cached full details of all nodes in the cluster
+   -->
+   此字段指示扩展模块可以缓存节点信息，从而调度器应该发送关于可选节点的最少信息，
+假定扩展模块已经缓存了集群中所有节点的全部详细信息。
+</td>
+</tr>
+<tr><td><code>managedResources</code><br/>
+<a href="#kubescheduler-config-k8s-io-v1beta2-ExtenderManagedResource"><code>[]ExtenderManagedResource</code></a>
+</td>
+<td>
+   <!--
+   ManagedResources is a list of extended resources that are managed by
+this extender.
+- A pod will be sent to the extender on the Filter, Prioritize and Bind
+  (if the extender is the binder) phases iff the pod requests at least
+  one of the extended resources in this list. If empty or unspecified,
+  all pods will be sent to this extender.
+- If IgnoredByScheduler is set to true for a resource, kube-scheduler
+  will skip checking the resource in predicates.
+   -->
+   <p><code>managedResources</code> 是一个由此扩展模块所管理的扩展资源的列表。</p>
+   <ul>
+     <li>如果某 Pod 请求了此列表中的至少一个扩展资源，则 Pod 会在 filter、
+prioritize 和 bind （如果扩展模块可以执行绑定操作）阶段被发送到该扩展模块。</li>
```

Comment:
missing translation：
"If empty or unspecified, all pods will be sent to this extender."